content/lessons/chapter-3/coop-2.tsx

```diff
@@ -35,19 +35,19 @@ export default function Coop2({ lang }) {
   const PROTAGONISTS = [
     {
       username: 'You',
-      avatar: '/assets/avatars/1.png',
+      avatar: account?.avatar,
```

Comment:
Do we need a fallback here in case avatar is not available, or is that handled elsewhere?

Reply:
the fallback for these are handled in the Profile and contributionBar components respectively